api/desecapi/pdns_change_tracker.py

```diff
@@ -31,6 +31,10 @@ class PDNSChangeTracker:
     `modifications` and `deletions` are guaranteed to be disjoint.
     - If an item is in the set of deletions while being modified, an exception is raised.
     - If an item is in the set of modifications while being deleted, it is removed from `rr_set_modifications`.
+
+    Note every change tracker object will track all changes to the model across threading.
+    To avoid side-effects, it is recommended to run only one change tracker at a time, i.e.
+    do not use them in parallel (in a multi-threading scenario), do not use them nested.
```

Comment:
Is the problem at hand really related to threading? If so, I don't understand what's happening: Our [uwsgi configuration](https://github.com/desec-io/desec-stack/blob/2c20d11/api/uwsgi.ini#L6) has `threads = 1`.

Reply:
It's not, but the note is still true